CorsixTH/Lua/config_finder.lua

```diff
@@ -488,6 +491,26 @@ audio_music = nil -- [[X:\ThemeHospital\Music]]
 --
 soundfont = nil -- [[X:\ThemeHospital\FluidR3.sf3]]
 
+-------------------------------------------------------------------------------
+-- Midi API and Device settings.
```

Comment:
I would be tempted to say this now crosses territory into an advanced setting. It is probably best most players don't even try to touch it.

Reply:
I suspect most players would be better off adjusting these in game; port is nearly impossible to get right otherwise, API is fairly simple though.